apps/backend/src/auth/roles.guard.ts

```diff
@@ -17,6 +17,6 @@ export class RolesGuard implements CanActivate {
       return true;
     }
     const { user } = context.switchToHttp().getRequest();
-    return requiredRoles.some((role) => user.role === role);
+    return requiredRoles.some((role) => user.role === role || user.role === Role.SUPERUSER);
```

Comment:
really nitpicky, but `user.role === Role.SUPERUSER || requiredRoles.some((role) => user.role === role)` is more efficient and readable